packages/react/src/components/Carousel/Carousel.tsx

```diff
@@ -90,6 +93,8 @@ const Carousel: FC<CarouselProps> & WithWrapperProps = (props: CarouselProps): R
 
   const classes: string = clsx('oxygen-carousel', className);
```

Comment:
```suggestion
  const classes: string = clsx('oxygen-carousel', {mobile: isMobile}, className);
```
This is the way we have previously added styling to mobile classes. Shall we select a consistent way?

Reply:
We can refer the `Header` or `Footer` components to see how we have already implemented styling for mobile screens.